p2p/protocols/protocol.go

```diff
@@ -336,6 +352,13 @@ func (p *Peer) handleIncoming(handle func(ctx context.Context, msg interface{})
 		return errorf(ErrDecode, "<= %v: %v", msg, err)
 	}
 
+	if p.spec.Hook != nil {
+		err := p.spec.Hook.Receive(p, wmsg.Size, val)
```

Comment:
to be honest, speaking of cheating, there are so many ways this can be cheated, that we should have a conversation about that specifically. Maybe in Prague.

Reply:
well the most simple and easy is editing your stateStore and change the balances

Reply:
how do you access my statestore??